packages/atlas-service/src/main.ts

```diff
@@ -64,21 +67,24 @@ export async function throwIfNotOk(
   }
 
   let serverErrorName = 'NetworkError';
+  let serverStatusCode = res.status;
   let serverErrorMessage = `${res.status} ${res.statusText}`;
   // We try to parse the response to see if the server returned any information
   // we can show a user.
   try {
     const messageJSON = await res.json();
     if (isServerError(messageJSON)) {
       serverErrorName = 'ServerError';
+      serverStatusCode = messageJSON.error;
```

Comment:
Why would network status code returned from the server not match the one in the error?

Reply:
This is what i try to understand as well.

Reply:
Removed. Probably was some flake with the local backend.